.gitignore

```diff
@@ -3,7 +3,7 @@
 *.iml
 .env
 .DS_Store
-venv/
+.venv/
```

Comment:
The `.venv/` directory is correctly listed to be ignored. However, ensure that this is the correct virtual environment directory you are using. If your virtual environment is named differently, you should update this line accordingly.